lib/DDG/Meta/Helper.pm

```diff
@@ -64,6 +65,17 @@ Use booleans true and false to set options.
 
     $stash->add_symbol('&true', sub { 1 });
     $stash->add_symbol('&false', sub { 0 });
+
+=keyword rand_int
+
+generates random integer between 0 and supplied max integer (max defaults to 1).
+
+=cut
+
+	$stash->add_symbol('&rand_int', sub { 	
+		my $max = shift || 1;
```

Comment:
@nilnilnil Could we not bound it ourselves and normalize for the package? That way users need know less about the underlying implementation when generating numbers within bounds.